src/main/java/ru/yandex/qatools/ashot/comparison/ImageDiffer.java

```diff
@@ -79,6 +85,10 @@ protected void markDiffPoints(Screenshot expected, Screenshot actual, ImageDiff
     }
 
     private boolean hasDiffInChannel(Screenshot expected, Screenshot actual, int i, int j) {
+        if(ignoredColor != null && rgbCompare(expected.getImage().getRGB(i, j), ignoredColor.getRGB(), 0)) {
```

Comment:
=) Are you sure?

If ignoredColor is null return will be:
return !rgbCompare(expected.getImage().getRGB(i, j), actual.getImage().getRGB(i, j), colorDistortion);

see row №92

P.S. All unit tests PASSED